src/main/java/com/example/zoospringv1/Mapper.java

```diff
@@ -0,0 +1,37 @@
+package com.example.zoospringv1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
```

Comment:
w spring jest adnotacja @Component, to domysla dzieki ktorej sprint zarządza twoja klasa przez Dependency Injection.

@Service robi dokladnie to samo, ale po prostu widac ze to serwis.

Takze tu moze byc po prostu @Component